src/ray/object_manager/common.h

```diff
@@ -8,7 +8,8 @@ namespace ray {
 /// A callback to asynchronously spill objects when space is needed. The
 /// callback returns the amount of space still needed after the spilling is
 /// complete.
-using SpillObjectsCallback = std::function<int64_t(int64_t num_bytes_required)>;
+using SpillObjectsCallback =
+    std::function<int64_t(int64_t num_bytes_required, int64_t min_bytes_required)>;
```

Comment:
```suggestion
    std::function<int64_t(int64_t num_bytes_to_spill, int64_t min_bytes_to_spill)>;
```

Can you also update the comment?